server/appengine/src/main/webapp/WEB-INF/web.xml

```diff
@@ -47,6 +38,6 @@
 
   <mime-mapping>
     <extension>yaml</extension>
-    <mime-type>application/x-yaml</mime-type>
+    <mime-type>application/x-yaml;charset=utf-8</mime-type>
```

Comment:
Sadly don't mime-type supports charset.  From earlier googleing I think we have to have a filter enforce the charset.

We're getting ` application/octet-stream` now
```
curl 'https://whoapp.org/content/bundles/your_questions_answered.en.yaml' -H 'authority: whoapp.org' -H 'pragma: no-cache' -H 'cache-control: no-cache' -H 'upgrade-insecure-requests: 1' -H 'user-agent: Mozilla/5.0 (Linux; Android 8.0.0; Pixel 2 XL Build/OPD1.170816.004) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.163 Mobile Safari/537.36' -H 'sec-fetch-dest: document' -H 'accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3;q=0.9' -H 'sec-fetch-site: none' -H 'sec-fetch-mode: navigate' -H 'sec-fetch-user: ?1' -H 'accept-language: en-US,en;q=0.9' --compressed -I
HTTP/2 200 
date: Thu, 16 Apr 2020 01:10:11 GMT
expires: Thu, 16 Apr 2020 01:20:11 GMT
etag: "6NoKKw"
content-type: application/octet-stream
server: Google Frontend
cache-control: public, max-age=600
content-length: 18317
age: 34